lib/spack/spack/test/verification.py

```diff
@@ -136,7 +136,7 @@ def test_check_prefix_manifest(tmpdir):
     prefix_path = tmpdir.join('prefix')
     prefix = str(prefix_path)
 
-    spec = spack.spec.Spec('libelf')
+    spec = spack.spec.Spec('libelf@0.8.13')
```

Comment:
Same question here: why was it necessary to add a version?

Reply:
This one didn't actually need a version.